index.html

```diff
@@ -206,10 +204,10 @@ <h3 class="ulheader">Skills</h3>
             action="https://formspree.io/f/xeqwogzg"
             method="POST">
               <div id="input_cont">
-                <input class="st-input" type="text" name="name" placeholder="Your name" maxlength="30" required>
-                <input class="st-input" type="email" name="email" placeholder="Email address" required>
-              </div>
-              <textarea name="message" type="text" maxlength="500" placeholder="Enter your message here..." required></textarea>
+                <label id="name_inp">Your name<input class="st-input" type="text" name="name" placeholder="Your name" maxlength="30" required></label>
+                <label id="email_inp">Your email<input class="st-input" type="email" name="email" placeholder="Email address" required></label>
+              </div> 
+              <label id="message_inp">Your message<textarea name="message" type="text" maxlength="500" placeholder="Enter your message here..." required></textarea></label>

```

Comment:
- [ ] Kindly consider adding a `for` attribute in your label that would have a corresponding `id` in its `input` due to follow best practices. please check out this link for more [info](https://www.w3schools.com/tags/tag_label.asp)